<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalLevels</class>
 <widget class="QDialog" name="DigitalLevels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="digLevel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digLevel2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="digLevel3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>Level 3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
